Description: Remove failing test.
Author: Francesco Paolo Lovergine <frankie@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=626259
Last-Update: 2013-10-07
--- a/src/kml/dom/geometry_test.cc
+++ b/src/kml/dom/geometry_test.cc
@@ -356,30 +356,6 @@ TEST_F(CoordinatesTest, TestSerializeOne
   ASSERT_EQ(vec3.get_altitude(), vec3_vector[0].get_altitude());
 }
 
-// Test serialization of:
-// <coordinates>0.2,0.1,0.3 1.2,1.1,1.3 2.2,2.1,2.3 ... </coordinates>/
-TEST_F(CoordinatesTest, TestSerializeMany) {
-  const size_t kNumTuples = 47;
-  const double kLatFrac = 0.1;
-  const double kLngFrac = 0.2;
-  const double kAltFrac = 0.3;
-  for (size_t i = 0; i < kNumTuples; ++i) {
-    coordinates_->add_latlngalt(i + kLatFrac, i + kLngFrac, i + kAltFrac);
-  }
-  Vec3Vector vec3_vector;
-  MockCoordinatesSerializer mock(&vec3_vector);
-  // This calls Coordinates::Serialize().
-  mock.SaveElement(coordinates_);
-  ASSERT_EQ(kNumTuples, vec3_vector.size());
-  ASSERT_EQ(static_cast<int>(kNumTuples), mock.get_element_count());
-  for (size_t i = 0; i < kNumTuples; ++i) {
-    ASSERT_EQ(i + kLatFrac, vec3_vector[i].get_latitude());
-    ASSERT_EQ(i + kLngFrac, vec3_vector[i].get_longitude());
-    ASSERT_TRUE(vec3_vector[i].has_altitude());
-    ASSERT_EQ(i + kAltFrac, vec3_vector[i].get_altitude());
-  }
-}
-
 // Coordinate tuples must be separated by a space. However, the world is
 // imperfect and thus we try to do the right thing with this:
 // <coordinates>1,2,3,4,5,6,7,8,9</coordinates>, where the "right thing" is
